src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -4667,8 +4670,6 @@ dup2(int oldfd, int newfd)
 			return fd;
 		else
 			return idx;
-	} else {
-		next_dup2(oldfd, newfd);
```

Comment:
just a minder this is on purpose right?

Reply:
Coverity will still complain dead code here. I will set ignoring this issue in coverity at this moment.

Reply:
ah i see. then nevermind adding the assert